closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -677,7 +667,7 @@ private static byte[] getEncodedKey(Key key) throws InvalidKeyException {
      */
     @Override
     protected byte[] engineUpdate(byte[] in, int inOfs, int inLen) {
-        byte[] out = new byte[inLen];
+        byte[] out = new byte[engine.getOutputSize(inLen, false)];
         try {
             int size = engine.doUpdate(in, inOfs, inLen, out, 0);
             // Special case for EngineAEADDec, doUpdate only buffers the input
```

Comment:
Can you please fix this comment: I think it's one sentence, so it should end with a period and the second line should not start with a capital.

Reply:
Fixed this sentence as suggested. I lower cased the comment on the second line and added a period.

Reply:
Added `has` to this sentence using [04f96c5](https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/744/commits/04f96c5d650a6ff8fdf18ea9d4a1ce2ce4095843)